helpers/compare.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
 const factory = () => {
-    return function(lvalue, rvalue) {
-        const options = arguments[arguments.length - 1];
+    return function(lvalue, rvalue, options) {
+
         var operator;
         var operators;
         var result;
 
-        if (arguments.length < 3) {
+        if (options === undefined) {
```

Comment:
The most robust way to check for undefined is `typeof x === 'undefined'`

Reply:
will change that.